frontend/src/index.jsx

```diff
@@ -1,21 +1,21 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
```

Comment:
Lets try to avoid moving around imports(avoid unnecessary changes that may cause git conflicts) in the future. Unless the linter has a specific reason for this ordering?

Reply:
airbnb has a specific ordering. Package imports before file imports.